ocaml/xapi/xapi_globs.ml

```diff
@@ -52,8 +52,8 @@ let xapi_user_agent = "xapi/"^(string_of_int version_major)^"."^(string_of_int v
  *
  * Please consult the XenCenter maintainers before changing these numbers, because a corresponding change
  * will need to be made in XenCenter *)
-let xencenter_min_verstring = "2.11"
-let xencenter_max_verstring = "2.11"
+let xencenter_min_verstring = "2.12"
+let xencenter_max_verstring = "2.12"
```

Comment:
Looks like we always bump these 2 together, and consider this is a platform release I think it is right.